server/validators/product/productValidation.js

```diff
@@ -0,0 +1,23 @@
+import Product from "../../models/product.js";
+
+export const validateCreate = async (fields) => {
+    const MAX_FILE_SIZE_MB = 2;
+    const { name,description,price,category,quantity,shipping,photo } = fields;
```

Comment:
Esta faltando o eslint configurado

```suggestion
    const { name, description, price, category, quantity, shipping, photo } = fields;
```